.github/workflows/publish.yaml

```diff
@@ -78,8 +78,8 @@ jobs:
       - name: Bump version and build
         if: ${{ steps.do-publish.outputs.publish }}
         run: |
-          git config --global user.email "jlem@tv2.dk"
-          git config --global user.name "leerberg"
+          git config --global user.email "info@superfly.tv"
```

Comment:
Should we be publishing with SuperFlys credentials? That doesn't seem right.
I do however agree that we shouldn't use Jesper Leerbegs credentials anymore

Reply:
I agree with this, but currently I don't have a better solution. I'd prefer if we can avoid using personal accounts and access tokens for this.